doc/md/privacy.md

```diff
@@ -469,51 +472,49 @@ func DenyMismatchedTenants() privacy.MutationRule {
 		}
 		// Query the tenant-id of all users. Expect to have exact 1 result,
 		// and it matches the tenant-id of the group above.
-		uid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
+		userTid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
 		if err != nil {
-			return privacy.Denyf("querying the tenant-id %w", err)
+			return privacy.Denyf("querying the tenant-id %v", err)
 		}
-		if uid != tid {
-			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, uid)
+		if userTid != tid {
+			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, userTid)
 		}
 		// Skip to the next privacy rule (equivalent to return nil).
 		return privacy.Skip
 	})
-	// Evaluate the mutation rule only on group creation.
-	return privacy.OnMutationOperation(rule, ent.OpCreate)
 }
 ```
 
 We add this rule to the `Group` schema and run code-generation.
 
-```go
+```go title="examples/privacytenant/ent/schema/group.go"
 // Policy defines the privacy policy of the Group.
 func (Group) Policy() ent.Policy {
 	return privacy.Policy{
 		Mutation: privacy.MutationPolicy{
-			rule.DenyMismatchedTenants(),
+			privacy.OnMutationOperation(
+				rule.DenyMismatchedTenants(),
+				ent.OpCreate),
```

Comment:
tbh, I prefer the previous version. Can you please explain why do you think it's needed?